cmd/forkooor.js

```diff
@@ -450,6 +455,35 @@ const mcdCloseStrategySub = async (vaultId, type, price, priceState, sender) =>
     console.log(`Subscribed to mcd close strategy with sub id #${subId}`);
 };
 
+const cbRebondSub = async (bondId, sender) => {
+    let senderAcc = (await hre.ethers.getSigners())[0];
+
+    await redeploy('CBRebondTrigger', REGISTRY_ADDR, false, true);
+    await redeploy('CBRebondSubProxy', REGISTRY_ADDR, false, true);
+    await redeploy('CBUpdateRebondSub', REGISTRY_ADDR, false, true);
+
+    if (sender) {
+        senderAcc = await hre.ethers.provider.getSigner(sender.toString());
+        // eslint-disable-next-line no-underscore-dangle
+        senderAcc.address = senderAcc._address;
+    }
+
+    let proxy = await getProxy(senderAcc.address);
+    proxy = sender ? proxy.connect(senderAcc) : proxy;
+
+    const cbRebondStrategy = createCbRebondStrategy();
+
+    await openStrategyAndBundleStorage(true);
+
+    let strategyId = await createStrategy(proxy, ...cbRebondStrategy, true);
```

Comment:
once strat is deployed we might want to comment this out, as it's not needed (strategyId hardcoded down, and this might mix up something in the future when testing on a fork)